### PR TITLE
Add rhelai instructlab image in 2.15 specific images

### PR DESCRIPTION
**Pull request by abhijeet-dhumal**
Additional images required for running distribiuted instructlab training on rhoai
Source : https://github.com/redhat-et/ilab-on-ocp/tree/main/standalone

### CODE REVIEW

**Review by szaher — APPROVED** [review]
lgtm

File: rhoai-disconnected-images.yaml [hunk 1]
```diff
@@ -4,4 +4,5 @@ additional-images:
 - quay.io/modh/ray@sha256:0d715f92570a2997381b7cafc0e224cfa25323f18b9545acfd23bc2b71576d06
 - quay.io/modh/ray@sha256:db667df1bc437a7b0965e8031e905d3ab04b86390d764d120e05ea5a5c18d1b4
 - quay.io/modh/ray@sha256:f8b4f2b1c954187753c1f5254f7bb6a4286cec5a4f1b43def7ef4e009f2d28cb
-
+- quay.io/opendatahub/workbench-images@sha256:7f26f5f2bec4184af15acd95f29b3450526c5c28c386b6cb694fbe82d71d0b41
```
Comment (sutaakar) [inline rhoai-disconnected-images.yaml]: This image should be replaced by one of productized Notebook images.

Comment (abhijeet-dhumal) [reply rhoai-disconnected-images.yaml]: I have tested one of the productized image for 2.15 release `quay.io/modh/odh-generic-data-science-notebook@sha256:16fc91984a9baf7765a6362493906a2c726b9906031711e2e55d686d296e6b3a` to run e2e ilab training, and it works as expected :heavy_check_mark:

Comment (abhijeet-dhumal) [reply rhoai-disconnected-images.yaml]: So I'll be removing this workbench image..
Thanks @sutaakar for reviewing this PR !!

**Review by sutaakar — APPROVED** [review]

**Pull request merged**